src/components/flag/Flag.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+import * as Flags from "./flags"
+import { SFlags } from "./styles/SFlags";
+
+const Flag: FC<any> = ({ countryCode, size }) => {
+  const Flag = Flags[countryCode]
```

Comment:
Country code should not be case sensitive. It should work the same way for 'AR' 'Ar' or 'ar'. Validate that the string passed as key is valid, it should be part of the interface, similar to the allowed icon sizes.